pkg/cloudprovider/providers/aws/cloudformation.go

```diff
@@ -186,17 +194,25 @@ func makeClusterInfraStackName(clusterName string) string {
 	return fmt.Sprintf("keto-%s-%s", clusterName, clusterInfraStackType)
 }
 
-func (c *Cloud) createMasterPoolStack(p model.MasterPool, infraStackName, amiID, elbName, kubeAPIURL string) error {
-	templateBody, err := renderMasterStackTemplate(p, infraStackName, amiID, elbName)
+func (c *Cloud) createMasterPoolStack(
+	p model.MasterPool,
+	infraStackName string,
+	amiID string,
+	elbName string,
+	kubeAPIURL string,
+	assetsBucketName string,
+) error {
+
+	templateBody, err := renderMasterStackTemplate(p, infraStackName, amiID, elbName, assetsBucketName)
 	if err != nil {
 		return err
 	}
 	stack := &cloudformation.CreateStackInput{
 		StackName: aws.String(makeMasterPoolStackName(p.ClusterName, "")),
 		Capabilities: aws.StringSlice([]string{
 			cloudformation.CapabilityCapabilityIam, cloudformation.CapabilityCapabilityNamedIam}),
-		Tags: makeStackTags(p.ClusterName,
-			masterPoolStackType, p.Name, p.KubeVersion, p.OSVersion, p.MachineType, kubeAPIURL, p.DiskSize),
+		Tags: makeStackTags(p.ClusterName, masterPoolStackType, p.Name, p.KubeVersion, p.OSVersion,
```

Comment:
Basically passing a `tagKey => value` map and handling it in `makeStackTags`. Basically right now every time a new tag gets added you need to update all invocations of `makeStackTags` to pass a value (or empty string).